nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -68,10 +68,13 @@ Make sure to also check the many updates in the [Nixpkgs library](#sec-release-2
   binary, but the previous build used a wrapper script that would store them in
   `~/.config/unity3d/srylain Inc_/Clone Hero`.
 
+
 - `services.mastodon` doesn't support providing a TCP port to its `streaming`
   component anymore, as upstream implemented parallelization by running
   multiple instances instead of running multiple processes in one instance.
-  Please create a PR if you are interested in this feature.
+  Please create a PR if you are interested in this feature.\
+  Due to this, the desired number of such instances
+  {option}`services.mastodon.streamingProcesses` now needs to be declared explicitly.
```

Comment:
```suggestion
  Please create a PR if you are interested in this feature.
  Due to this, the desired number of such instances {option}`services.mastodon.streamingProcesses` now needs to be declared explicitly.
```
we don't need to squick this onto one line.

Reply:
> I just noticed that someone had trimmed the release notes markdown line length, and thus did the same.

Same and I totally disagree with that since then I can't easily read the changelog anymore and it is fine to use more than 1/6 of my screen width.